src/style.css

```diff
@@ -2,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.rating-star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, if you have a custom font for `.rating-star`, ensure you have a fallback like `sans-serif`.